tests/tracer/test_encoders.py

```diff
@@ -547,23 +547,28 @@ def filter_mut(ts):
     ]
 
 
-def string_table_test(t, offset=0):
-    assert len(t) == 1 + offset
+def string_table_test(t, origin_key=False):
+    assert len(t) == 1 + origin_key
```

Comment:
.... running with `-O` doesn't remove these asserts right? 🤔

Reply:
This is the warning from `pytest`
~~~
PytestConfigWarning: assertions not in test modules or plugins will be ignored because assert statements are not executed by the underlying Python interpreter (are you using python -O?)
~~~

Reply:
ok, so it is still doing it's rewriting before execution so we should still be good ?

Reply:
I believe so